src/library_webgpu.js

```diff
@@ -2576,15 +2576,25 @@ var LibraryWebGPU = {
       {{{ gpu.makeGetU32('descriptor', C_STRUCTS.WGPUSwapChainDescriptor.presentMode) }}});
 #endif
 
+    var canvasSize = [
+        {{{ gpu.makeGetU32('descriptor', C_STRUCTS.WGPUSwapChainDescriptor.width) }}},
+        {{{ gpu.makeGetU32('descriptor', C_STRUCTS.WGPUSwapChainDescriptor.height) }}}
+    ];
+
+    if (canvasSize[0] !== 0) {
+        context.canvas.width = canvasSize[0];
```

Comment:
Use this to indicate to Closure compiler that it shouldn't try to minify these properties. I'm never sure if this is necessary, but I think it can't hurt.

```suggestion
        context["canvas"]["width"] = canvasSize[0];
```